mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -512,7 +512,7 @@ async def transformation(request: Request):
 
         data = await request.body()
         content_type = request.headers.get("content-type")
-        # TODO: convert "invocations" to an async method to make iternal logic fully non-blocking.
+        # TODO: convert "invocations" to an async method to make internal logic fully non-blocking.
```

Comment:
typo got merged? How? @harupy

Reply:
`typos` only checks words in its dictionary. I think `iternal` is not included. Let's add it.

Reply:
Filed https://github.com/mlflow/mlflow/issues/14455